docs/OrganizingCommittee/Committee-formation.md

```diff
@@ -0,0 +1,84 @@
+
+Here are some of general principles for recruiting a diverse and engaged organizing committee:
+
+* Designate one person or a small committee to be responsible for recruitment. Among this group, build a list of potential recruitees from each person's network of trusted colleagues. This list may also contain names of people who have approached the hack week organizers with an intention to participate. This is common for hack weeks that span multiple years, where past participants wish to be more engaged in subsequent events. 
+* For recruitees who are completely new to the group and who have not experienced a hack week, initiate a 30 minute meeting with the lead coordinator(s). Begin the meeting with a review of the hack week mission and ask the recruitee to offer some reflections on how their participation would serve to support that mission. This meeting is also an opportunity to share ideas on potential ways the recruitee can get involved.
+* An e-mail invitation to both new and continuing potential committee members is an effective way to establish an organizational structure early in the process. The purpose of the message is to:
+    * thank the pontential committee member for considering this activity
+    * provide links to logistics information (event location, timing, travel arrangements)
+    * provide a reminder of the roles, responsibilities and potential time commitment for this activity
+    * offer to have this person's information included on the hack week website
+* For some hack weeks we have used the participant application form as an efficient way to initially assemble an organizing committee. Committee members will be asked to fill out this form anyhow, for the sake of assessing our overall team diversity. In this process we also ask them to provide details on the open-ended questions about how they would like to be involved on an administrative level. 
+
+# Recognizing contributions from committee memebers
+
+Hack week organizing committees often include a mix of people who are both funded to participate in the hack week activities, and who are contributing their time voluntarily. In most cases, hack weeks would not happen without the generosity of many people who believe in the community building and educational mission of these events. In order to sustain the continued offering of hack weeks, it is important to find ways to acknowledge and if possible, reimburse people for their time and effort. 
+
+Here are some ideas for recognizing people's contributions to hack weeks:
+
+* ask how individuals would like to be recognized: some people want to be paid (e.g. more senior faculty who are not allowed to charge an existing grant for this type of work), others (e.g. postdocs) would rather have some form of formal recognition they can point to for their resume. 
+* provide payment: grant to charge to for local people, honorarium for others

```

Comment:
I'm not quite sure I understand the second part of this bullet point. Maybe expand a bit?